app/models/school_cohort.rb

```diff
@@ -59,10 +59,22 @@ def lead_provider
     school.lead_provider(cohort.start_year)
   end
 
+  def previous_lead_provider
+    school.lead_provider(cohort.start_year - 1)
+  end
+
```

Comment:
Is this used anywhere? If it is could we move this somewhere else please, or remove if not?

Reply:
This is used in the DashBoardController to show the previous lead provider when a partnership is not present.
I have put this method in the SchoolCohort class because it is related to the School and Cohort to which the SchoolCohort belongs to.
I'm happy to move it somewhere else. Any idea?

Reply:
I would probably handle this in the DashboardController, perhaps just find the previous school_cohort and then call the `lead_provider` method on that, if that makes sense?  Just feels like navigating different cohorts and providers should be on the model?